### ISSUE TITLE
COMPILE ROOT CONFIGURATION error

### ISSUE THREAD

**Issue by FLeven** [gaelcolas/DscInfraSample#22]
.Build.ps1 -ResolveDependency

runs fine, until COMPILE ROOT CONFIGURATION

SRV01 : SRV01 : MOF__70b45ea Dev
PSDesiredStateConfiguration\node : In C:\Users\paul\Documents\git\DscInfraSample\DSC_Resources\xPSDesiredStateConfigura
tion\8.2.0.0\DSCPullServerSetup\PublishModulesAndMofsToPullServer.psm1:43 Zeichen:16
+     Log -Scope $MyInvocation -Message 'Start Deployment'
+                ~~~~~~~~~~~~~

and many more erros all related to xPSDesiredStateConfiguration

SRV02 : SRV02 : MOF__70b45ea Dev
    SRV02 : WindowsBase ::> Shared1 ....................................................................................OK
    SRV02 : WindowsBase ::> SoftwareBase ...............................................................................OK
ERROR OCCURED DURING COMPILATION

C:\Windows\System32\WindowsPowerShell\v1.0\Modules\PSDesiredStateConfiguration\PSDesiredStateConfiguration.psm1:3917
Zeichen:5
+     throw $ErrorRecord
+     ~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (RootConfiguration:String) [], InvalidOperationException
    + FullyQualifiedErrorId : FailToProcessConfiguration

and another error about Get-DscSplattedResource could not be loaded.

ROOT META MOF compiles woithout any error.

**Comment by raandree** [comment]
That seems to be the same issue happening in AutomatedLab/DscWorkshop#30.

**Comment by gaelcolas** [comment]
Thanks @FLeven for reporting this.
Although the AutomatedLab/DscWorkshop project is a bit more complex, I tend to treat it as more up-to-date than DSC infra sample. Although not perfect, it has a few more features, better build layout and error management.

For the basics info and layout, DscInfraSample is still relevant and may have more docs, but I'd recommend also looking at DscWorkshop.

Anyway, this is now fixed and I've updated the project to use the latest Chocolatey module version, and updated the gaelcolas/sharedDscConfig project.

Let me know of any issue.
